USFMConverter/Core/Render/RenderDocument.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using USFMConverter.Core.ConstantValue;
+using USFMConverter.Core.Data;
+using USFMToolsSharp;
+using USFMToolsSharp.Models.Markers;
+
+namespace USFMConverter.Core.Render
+{
+    public abstract class RenderDocument
+    {
+        public RenderDocument()
+        {
+            
+        }
+
+        /// <summary>
+        /// Parses the given text files into one USFM Document.
+        /// </summary>
+        /// <param name="files">Text files with USFM format.</param>
+        /// <returns>A USFM Document</returns>
+        public USFMDocument LoadUSFMs(IEnumerable<FileInfo> files)
```

Comment:
These functions don't belong here -- they have nothing specifically to do with a RenderableDocument.  They should either be in a utility class such as USFMUtils, or perhaps in the FileSystem interface.